internal/envoy/v3/stats.go

```diff
@@ -21,6 +21,7 @@ import (
 	http "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 	envoy_tls_v3 "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/wellknown"
+	"github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
this package has been superceded by [google.golang.org/protobuf](https://pkg.go.dev/mod/google.golang.org/protobuf) which is already included in this file, so we should use that instead